internal/testrunner/testrunner.go

```diff
@@ -55,12 +76,12 @@ func RegisterRunner(testType TestType, runFunc RunFunc) {
 }
 
 // Run method delegates execution to the registered test runner, based on the test type.
-func Run(testType TestType, testFolderPath string) error {
+func Run(testType TestType, testFolder TestFolder, packageRootPath string) error {
```

Comment:
I think we can sync up both branches.

I introduced a similar concept of `TestOptions`: https://github.com/elastic/elastic-package/blob/c3fd692e9923f4160b11b3eac569b76014552934/internal/testrunner/testrunner.go to easily add more arguments here.